packages/console/src/pages/SignInExperience/components/ManageLanguage/LanguageEditor.tsx

```diff
@@ -1,27 +1,29 @@
-import { LanguageTag } from '@logto/language-kit';
 import { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 import Modal from 'react-modal';
 
 import ConfirmModal from '@/components/ConfirmModal';
 import ModalLayout from '@/components/ModalLayout';
+import useUiLanguages from '@/hooks/use-ui-languages';
 import * as modalStyles from '@/scss/modal.module.scss';
 
-import { LanguageEditorContext } from '../../hooks/use-language-editor-context';
-import LanguageEditor from './LanguageEditor';
+import useLanguageEditorContext, {
+  LanguageEditorContext,
+} from '../../hooks/use-language-editor-context';
```

Comment:
the hook can be moved to the component folder?

Reply:
updated.